sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/models/StorageAccount.java

```diff
@@ -547,6 +561,29 @@ interface WithHns {
             WithCreate withHnsEnabled(boolean enabled);
         }
 
+        /** The stage of storage account definition allowing to configure allow cross tenant replication. */
+        interface WithAllowCrossTenantReplication {
+            /**
+             * Disables allow cross tenant replication
+             *
+             * Disabling in storage account overrides the allow cross tenant replication settings for individual containers.
+             *
+             * @return the next stage of storage account definition
+             */
+            WithCreate disableAllowCrossTenantReplication();
```

Comment:
nit, not sure `disableAllowCrossTenantReplication` or `disableCrossTenantReplication`.

Reply:
From the meaning of `allowCrossTenantReplication`, this is an allow setting, so I think the `disableAllowCrossTenantReplication` is more appropriate.

Reply:
Agree, `disableCrossTenantReplication` is better. `disable`(`enable`) and `allow` seem duplicate.

Reply:
`enableAllowCrossTenantReplication` rename to `enableCrossTenantReplication`
`disableAllowCrossTenantReplication` rename to `disableCrossTenantReplication`
Is that right ?

Reply:
Let us check more doc. @v-hongli1 If you have doc from learn.microsoft.com, would you paste the link here?

Reply:
"enableAllow*" has the problem of having 2 verb.

But if we remove the "Allow" it may look like we always do cross tenant replication, while this may not be the exact case.

Reply:
Or perhaps, `allowCrossTenantReplication` and `disallowCrossTenantReplication`.

Reply:
Yeah, that looks good.

Let's wait a bit for Hong's doc link,

Reply:
I haven't found any documentation yet, but I have found similar code examples. I don't know if they can be used as a reference.
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/implementation/StorageAccountImpl.java#L529

Reply:
No. Not Java code.

I mean service doc about the feature e.g. https://learn.microsoft.com/en-us/azure/storage/blobs/object-replication-prevent-cross-tenant-policies?tabs=azure-cli

Let's use Xiaofei's suggestion above. https://github.com/Azure/azure-sdk-for-java/pull/37460#discussion_r1378359912

Reply:
`enableAllowCrossTenantReplication` rename to `allowCrossTenantReplication` and`disableAllowCrossTenantReplication` rename to `disallowCrossTenantReplication` in the new version.